app/models/concerns/image_size_validation.rb

```diff
@@ -0,0 +1,9 @@
+module ImageSizeValidation
+  extend ActiveSupport::Concern
+
+  def image_size_restriction
+    if image&.size > 5.megabytes
```

Comment:
`&.` I don't think that can ever be nil, can it? i think it always returns the Carrierwave mounted object, whether there's a file or not.